app/src/test/java/org/mozilla/focus/MainActivityViewModelTest.kt

```diff
@@ -6,14 +6,21 @@ package org.mozilla.focus
 
 import android.arch.core.executor.testing.InstantTaskExecutorRule
 import android.arch.lifecycle.MutableLiveData
+import kotlinx.coroutines.ObsoleteCoroutinesApi
+import kotlinx.coroutines.test.TestCoroutineContext
+import kotlinx.coroutines.test.withTestContext
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.mockito.Mockito.`when`
 import org.mockito.Mockito.mock
 import org.mozilla.focus.helpers.ext.assertValues
 import org.mozilla.focus.session.SessionRepo
+import java.util.concurrent.TimeUnit
 
+private const val EXIT_FULLSCREEN_EXCEED_EMISSION_DELAY_SECONDS = 6L
```

Comment:
Does this have to be larger than the actual delay?

Reply:
If not, it would be nice if the actual delay value were broken into a VisibleForTesting constant, and we relied on that in the test.  It would make them slightly less brittle.